pkgs/top-level/haskell-packages.nix

```diff
@@ -1,6 +1,16 @@
 { pkgs, callPackage, stdenv, buildPlatform, targetPlatform }:
 
-rec {
+let # These are the GHC versions that support building with integer-simple.
```

Comment:
Very cool, thank you. Added to `haskell-updates` in a7fffbf808572d1afd40a10f644eb213116a3bad.